File: tourneyClient/src/models/entities/Characters/YoungLink.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const YoungLink: Character =
{
    id: CharacterId.YoungLink,
    characterName: CharacterName.YOUNG_LINK,
    archetype: Archetype.TURTLE,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.LIGHTWEIGHT,
    tierPlacement: TierPlacement.B
}

export default YoungLink; 

File: tourneyAPI/Models/Entities/Characters/YoungLink.cs


namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class YoungLink : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public YoungLink()
    {
        Id = CharacterId.YoungLink;
        characterName = CharacterName.YOUNG_LINK;
        archetype = Archetype.TURTLE;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.LIGHTWEIGHT;
        tierPlacement = TierPlacement.B;
    }

}